source/MapDetailPanel.cpp

```diff
@@ -587,17 +589,17 @@ void MapDetailPanel::DrawKey()
 	{
 		// Each system is colored by the government of the system. Only the
 		// four largest visible governments are labeled in the legend.
-		vector<pair<double, const Government *>> distances;
-		for(const auto &it : closeGovernments)
+		vector<pair<unsigned int, const Government *>> screenNum;
```

Comment:
Isn't this effectively the same as governmentCounts?

Reply:
That's what it was doing before, my understanding is that is was to enable sorting by value

Reply:
ah ok